Dockerfile

```diff
@@ -0,0 +1,32 @@
+# Build Stage
```

Comment:
Plesae do not use git clone or `ldd`. Also, I notice you've been adding `version.json` in a lot of your submissions - can you explain this?

Reply:
Sure, I was using that when I thought git cloning my directory was the "cleanest" way of doing this. This would speed up my testing as building the Docker image would only invalidate the github clone layer and redownload when that file would change (Which would only happen on a new commit).

In short, it prevented performing a git clone every build when not needed